packages/victory-native/package.json

```diff
@@ -0,0 +1,61 @@
+{
+  "name": "victory-native",
+  "version": "36.0.0",
+  "description": "Native Port for Victory",
+  "keywords": [
+    "TODO:",
+    "data visualization",
+    "React",
+    "d3",
+    "charting"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/formidablelabs/victory.git"
+  },
+  "homepage": "https://formidable.com/open-source/victory",
+  "source": "src/index.js",
+  "main": "src/index.js",
+  "files": ["src", "index.js"],
+  "author": "Formidable",
+  "license": "MIT",
+  "dependencies": {
+    "victory": "^36.0.0",
+    "victory-area": "^36.0.0",
+    "victory-axis": "^36.0.0",
+    "victory-bar": "^36.0.0",
+    "victory-box-plot": "^36.0.0",
+    "victory-brush-container": "^36.0.0",
+    "victory-brush-line": "^36.0.0",
+    "victory-candlestick": "^36.0.0",
+    "victory-chart": "^36.0.0",
+    "victory-core": "^36.0.0",
+    "victory-create-container": "^36.0.0",
+    "victory-cursor-container": "^36.0.0",
+    "victory-errorbar": "^36.0.0",
+    "victory-group": "^36.0.0",
+    "victory-histogram": "^36.0.0",
+    "victory-legend": "^36.0.0",
+    "victory-line": "^36.0.0",
+    "victory-pie": "^36.0.0",
+    "victory-polar-axis": "^36.0.0",
+    "victory-scatter": "^36.0.0",
+    "victory-selection-container": "^36.0.0",
+    "victory-shared-events": "^36.0.0",
+    "victory-stack": "^36.0.0",
+    "victory-tooltip": "^36.0.0",
+    "victory-voronoi": "^36.0.0",
+    "victory-voronoi-container": "^36.0.0",
+    "victory-zoom-container": "^36.0.0"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.15.5",
+    "react-native": "^0.65.1",
+    "react-native-svg": "^12.1.1",
+    "react-native-gesture-handler": "^1.10.3"
+  },
+  "scripts": {
+    "version": "yarn run nps build-libs && nps build-dists"
```

Comment:
We're probably not going to build these when publishing native. We publish unaltered src only at the moment and I imagine that will remain so